report-service/src/main/java/com/overops/report/service/model/EventVisualizationModel.java

```diff
@@ -0,0 +1,93 @@
+package com.overops.report.service.model;
+
+import com.takipi.api.client.util.cicd.OOReportEvent;
+
+public class EventVisualizationModel {
+
+    private String arcLink;
+    private String type;
+    private String applications;
+    private String introducedBy;
+    private String eventSummary;
+    private String eventRate;
+    private long hits;
+    private long calls;
```

Comment:
What are `calls` ? I'm not sure this is ever used anywhere.

Reply:
Removed